banner/index.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <meta property="og:url" content="http://www.etcher.io" />
+    <meta property="og:type" content="website" />
+    <meta property="og:title" content="Etcher by Resin.io" />
+    <meta property="og:description" content="Burn images to SD cards and USB drives, safe and easy." />
+    <meta property="og:image" content="../static/images/interface.png" />
+    <link rel="shortcut icon" type="image/png" href="../static/images/etcher.ico"/>
+    <!-- The above 3 meta tags *must* come first in the head; any other head content must come *after* these tags -->
+    <title>Etcher by resin.io</title>
+    <!-- Bootstrap -->
+    <link href="../static/styles/main.css?v=1.5" rel="stylesheet">
+    <!-- fonts -->
+    <script src="https://use.typekit.net/lzw7tre.js"></script>
```

Comment:
@Shou could we make this _not_ depend on 3rd-party sites? Or would that break the styling?